### DISCUSSION TITLE
ReOPT not happy with negative numbers

### DISCUSSION THREAD

**Discussion by hmurphy02** [NREL/REopt-API-Analysis#160] [1 - General REopt Questions]
Hi all,

I've been working with reopt to help with a school, we were given all the data we need, but a lot of it is negative numbers. After trying to plug in the negative numbers to reopt, it wont seem to run, but when i take them out, it runs. the negative numbers are essential in getting a correct size system for the school. Anyone help?

**Comment by adfarth** [comment]
Hi, REopt currently does not accept negative load values. If these negative values are because there is existing PV, you might consider backing out the PV generation, creating a "gross load" (rather than net load) profile, and using the "existing PV" input to indicate that there is existing PV. If you enter the gross load + existing PV, be sure to set `PV.loads_kw_is_net` to `false`
